mmdet/models/dense_heads/anchor_head.py

```diff
@@ -52,6 +52,8 @@ def __init__(self,
                      loss_weight=1.0),
                  loss_bbox=dict(
                      type='SmoothL1Loss', beta=1.0 / 9.0, loss_weight=1.0),
+                 loss_normalizer=-1,
```

Comment:
The meaning of `loss_normalizer` and `loss_normalizer_momentum` should be reflected in docstring.